c++/src/kj/filesystem-disk-qnx.c++

```diff
@@ -0,0 +1,1472 @@
+// Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
```

Comment:
Note that his is largely cut and pasted from `filesystem-disk-unix.c++`, see what I wrote in the PR summary